documentation/book/ref-storage-jbod.adoc

```diff
@@ -5,7 +5,7 @@
 [id='ref-jbod-storage-{context}']
 = JBOD storage overview
 
-You can configure {ProductName} to use JBOD, a data storage configuration in which multiple disks are combined to create a single logical disk, or volume. JBOD is one approach to providing increased data storage for Kafka brokers. It can also improve performance.
+You can configure {ProductName} to use JBOD, a data storage configuration of multiple disks, or volume. JBOD is one approach to providing increased data storage for Kafka brokers. It can also improve performance.
```

Comment:
```suggestion
You can configure {ProductName} to use JBOD, a data storage configuration of multiple disks or volumes. JBOD is one approach to providing increased data storage for Kafka brokers. It can also improve performance.
```

Reply:
I read it as the commas being used to isolate a description of JBOD and volumes being another option. Will update.